ksqldb-functional-tests/src/test/resources/query-validation-tests/as_value.json

```diff
@@ -149,6 +149,27 @@
           {"name": "OUTPUT", "type": "stream", "schema": "ID DECIMAL(4,2) KEY, ID_COPY DECIMAL(4,2), V1 INT"}
         ]
       }
+    },
+    {
+      "name": "STRUCT",
```

Comment:
Can we also add array and map for completeness? Same for the other tests (group by, partition by, etc). I don't think the coverage needs to be as thorough as what you've added for struct but I still think it'd be good to cover our bases.